Wally120/Arduino/Wally120_WIP/Wally120_WIP.ino

```diff
@@ -1,5 +1,5 @@
   //stepper
-#define speedRotation 1000  //speed of the motor rotation
+#define speedRotation 500  //speed of the motor rotation
```

Comment:
Because, now we use it for half a step instead of one whole step, see 421fd20034282814f227e9c3466ccb5a4af2fc44. There is no need to document this, as there are no units to this number, i.e. it's only a relative value.